tools/accuracy_checker/accuracy_checker/config/config_reader.py

```diff
@@ -784,6 +787,12 @@ def process_launchers(launchers_configs):
 
 
 def merge_entry_paths(keys, value, args, value_id=0):
+    def select_argument_by_priority(argument_list):
+        for current_arg in argument_list:
+            if current_arg in args and args[current_arg]:
+                return current_arg
+        return argument_list[-1]
```

Comment:
I would suggest `return next(filter(args.get, argument_list), argument_list[-1])`. It's simple enough that you might not even need a function.